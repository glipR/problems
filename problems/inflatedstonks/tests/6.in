15
LULDUULLRDLRUUU
UDRRULLUUDLLUDU
RDLLRULURLDRDLD
UDUDLRRLDUDDDLU
URRRDDUUDRLLDUL
RUDLRDRRRUUUUUL
DRDLLULDULULRLR
DRLUULURRRDLURD
LDUUDUUDURLRUDR
RLDDUDLUDURUDDR
RLLDURRUDLUURRD
DRRLLDUDLDDDDDL
ULRLLUULDDLLRUR
UUUULUUDULRDDDD
ULLRLRULDDLLUUD
61882 613465 223139 668536 888335 91975 339764 843912 657802 610018 739473 225667 239608 143572 535702
869566 604628 314624 776603 624873 184228 969675 994982 875344 280509 117514 95494 102708 297051 149751
671557 636321 658400 314426 301976 779616 739075 863140 305956 705915 491591 520396 669499 765565 494757
697105 393896 87466 843327 87119 188239 592963 511609 398161 986610 679075 170067 597461 921502 784477
261553 683001 908721 661667 649051 347650 277222 520046 714563 740145 938875 935973 280370 544604 85077
374792 578479 627977 381658 264011 404767 928384 620767 962939 863508 108942 847249 161898 789626 321193
566860 602599 359681 966577 57412 120671 415519 600047 210979 488950 901190 21932 156736 354754 61445
923152 466014 260405 777258 927399 117107 280855 47878 540032 446228 699224 242499 58841 654863 885059
666666 675833 409198 302555 411877 727717 657368 269674 148309 840697 200715 47211 933363 719786 919821
795958 670633 761960 168736 902288 113197 57850 137060 103613 475690 102701 595625 700163 382366 913660
580740 951138 134525 998976 781843 2527 397700 378818 517157 87998 614683 26490 526457 587186 560907
418255 669944 198624 206907 601104 178413 411412 632770 65505 25106 379540 135994 148255 118458 345423
94189 188524 345845 239906 363544 425930 732851 481663 79500 521055 447639 22907 575783 862488 80392
614535 723558 116274 668610 745236 668354 204416 667946 387568 573143 539099 224101 549408 701151 400576
170127 992820 123288 592983 503421 226865 860127 49085 214581 632693 305626 443409 725566 524466 814411