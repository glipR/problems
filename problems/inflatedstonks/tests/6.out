84
1 0
0 2
2 6
2 4
1 6
0 3
1 2
2 7
0 11
1 12
0 13
1 14
1 8
0 10
1 11
3 9
2 14
4 7
3 5
4 6
5 6
3 8
6 9
2 10
5 11
3 11
6 11
5 14
4 14
1 13
2 11
4 0
1 1
2 3
3 2
5 0
3 4
4 5
8 6
10 10
10 11
6 14
8 5
8 3
5 3
6 1
8 2
6 0
10 4
9 8
10 9
13 7
14 5
9 6
6 7
8 8
7 11
9 9
7 13
8 1
9 14
9 2
14 2
13 1
9 3
11 1
11 4
13 5
9 12
8 13
11 12
13 4
12 4
13 2
12 14
11 11
11 10
11 9
13 12
13 13
14 3
14 8
13 10
13 14
